flake.nix

```diff
@@ -13,17 +13,26 @@
     ocaml-overlays.url = "github:anmonteiro/nix-overlays";
     ocaml-overlays.inputs.nixpkgs.follows = "nixpkgs";
 
+    prometheus-web.url =
+      "github:marigold-dev/prometheus-web/ulrikstrid/nix-stuff";
+    prometheus-web.inputs.nixpkgs.follows = "nixpkgs";
+    prometheus-web.inputs.ocaml-overlay.follows = "ocaml-overlays";
+
     nix-npm-buildpackage.url = "github:serokell/nix-npm-buildpackage";
   };
 
-  outputs =
-    { self, nixpkgs, flake-utils, nix-npm-buildpackage, ocaml-overlays }:
+  outputs = { self, nixpkgs, flake-utils, nix-npm-buildpackage, ocaml-overlays
+    , prometheus-web }:
     with flake-utils.lib;
-    eachSystem [ system.x86_64-linux ] (system:
+    eachSystem defaultSystems (system:
```

Comment:
This doesn't really belong in this PR but it expands to

It's basically the same as
mac_x86
mac_aarch64
linux_x86
linux_aarch64